src/ctap/config_command.rs

```diff
@@ -0,0 +1,271 @@
+// Copyright 2020 Google LLC
```

Comment:
You need to add 2021 now :)

Reply:
Done, for this and all other files in CTAP.